### ISSUE TITLE
[expo-Image] view expo Image as an ImageBackground tag. But when nesting expo-Image, the background image is hidden outside

### ISSUE THREAD

**Issue by huytdps13400** [expo/expo#22338]
### Summary

[expo-Image] view expo Image as an ImageBackground tag. But when nesting expo-Image, the background image is hidden outside
![image](https://user-images.githubusercontent.com/68500260/235416352-40245f42-90ef-47f4-a957-7424c1269063.png)

### What platform(s) does this occur on?

Android

### SDK Version

1.1.0

### Environment

PROD

### Minimal reproducible example

![image](https://user-images.githubusercontent.com/68500260/235416390-52049152-5711-4446-8f70-386dc5a0fdfb.png)

**Comment by alanjhughes** [comment]
@huytdps13400 Image does not support children. You need to create this effect yourself with absolute positioning

**Comment by feRpicoral** [comment]
I'm not sure if it's intended or not, but it does support children, only on iOS (at least is working fine for me on `expo-image@1.0.1`. For Android we need the absolute positioning as @alanjhughes mentioned.

**Comment by feRpicoral** [comment]
@huytdps13400 I created this component to get rid of this problem:

Disclaimer: I haven't tested it that much, but for my use cases it seems to be working

```tsx
import { Image, ImageProps } from 'expo-image';
import React from 'react';
import { Platform, StyleSheet, View } from 'react-native';

/**
 * Wraper around `expo-image` to be able to use it with children on android.
 * @see https://github.com/expo/expo/issues/22338
 */
export function ImageBackground({ children, ...props }: ImageProps) {
    const isAndroid = Platform.OS === 'android';

    if (isAndroid) {
        return (
            <View style={styles.container}>
                <Image {...props} />
                <View
                    style={[
                        styles.chlidren,
                        StyleSheet.absoluteFill,
                        props.style
                    ]}
                >
                    {children}
                </View>
            </View>
        );
    }

    return <Image {...props} children={children} />;
}

const styles = StyleSheet.create({
    container: {
        position: 'relative'
    },
    chlidren: {
        zIndex: 1
    }
});

```

**Comment by alanjhughes** [comment]
@feRpicoral - the latest version of `expo-image` now exports a `ImageBackground` component

**Comment by feRpicoral** [comment]
@alanjhughes Which version exactly? I'm on `1.2.3` and couldn't find that mentioned on the docs: https://docs.expo.dev/versions/latest/sdk/image/

I'm aware of `react-native`'s `ImageBackground` but it doesn't have the same features as `expo-image`

**Comment by alanjhughes** [comment]
@feRpicoral - 1.3.0 it's a recent addition. I'm not sure if it is in the docs yet

**Comment by feRpicoral** [comment]
@alanjhughes It's not present in the SDK 48 branch: https://github.com/expo/expo/tree/sdk-48/packages/expo-image
I also tried searching for branches related to `expo-image` but from the two that showed up, none had this functionality.

**Comment by alanjhughes** [comment]
@feRpicoral - libraries can be updated independent of the sdk release https://github.com/expo/expo/blob/main/packages/expo-image/CHANGELOG.md

**Comment by feRpicoral** [comment]
@alanjhughes thanks for the link! Apparently, it's not released yet, as that's in the `1.3.0` changelog and the latest version on npm is `1.2.3`

**Comment by dr-thca** [comment]
> I'm not sure if it's intended or not, but it does support children, only on iOS (at least is working fine for me on `expo-image@1.0.1`. For Android we need the absolute positioning as @alanjhughes mentioned.

Any reason for this discrepancy between iOS and android? Our team was stuck on this for a couple of hours.
